doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1055,32 +1031,37 @@ of interpretations within a scope).
 
 .. cmd:: Open Scope @scope
 
-   The command to add a scope to the interpretation scope stack is
-   :n:`Open Scope @scope`.
+   Adds a scope to the interpretation scope stack.
+
+   When the command appears in a section, the scope is only added within the section.
+   When the command is outside a section, the scope is exported as part of the current module.
+
+   This command supports the :attr:`local` and :attr:`global` attributes.
+   :attr:`local` adds the scope only in the current module.  :attr:`global`
+   adds the scope outside the current section but has no effect if the
+   command is not inside a section.
 
 .. cmd:: Close Scope @scope
 
-   It is also possible to remove a scope from the interpretation scope
-   stack by using the command :n:`Close Scope @scope`.
+   Removes a scope from the interpretation scope stack.
 
    Notice that this command does not only cancel the last :n:`Open Scope @scope`
```

Comment:
Perhaps "note" would be a bit more idiomatic than "notice"

Reply:
Sure.  The goal of this PR is to get the correct syntax in the syntax extensions chapter and make necessary adjustments to reflect that syntax.  Very selectively rewriting some things.  And a few other changes that creep in.  There will be further passes through the document for wording, to add examples, introductions, explanation of some of the syntax and conceptual material.